<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsGuiComponent</class>
 <widget class="QFrame" name="CSettingsGuiComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI general settings</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="vl_SettingsGuiComponent">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_Font">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QVBoxLayout" name="vl_FontSettings">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="swift::gui::components::CSettingsFontComponent" name="comp_SettingsFonts">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_GuiGeneral">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QFormLayout" name="fl_GuiGeneral">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_SettingsGuiOpacity">
        <property name="toolTip">
         <string>0-100%</string>
        </property>
        <property name="text">
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="hs_SettingsGuiOpacity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>0-100%</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_SettingsGuiWidgetStyle">
        <property name="text">
         <string>Widget style:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cb_SettingsGuiWidgetStyle"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_PreferredMultiSelect">
        <property name="text">
         <string>Selection:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QWidget" name="wi_PreferredSelection" native="true">
        <layout class="QHBoxLayout" name="hl_PreferredMultiSelect">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="rb_PreferExtendedSelection">
           <property name="text">
            <string>Extended</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_PreferMultiSelection">
           <property name="text">
            <string>Multi</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hs_PreferredSelection">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>30</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CSettingsFontComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/settingsfontcomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
